ChatSecure/Classes/Controllers/OTRYapViewHandler.swift

```diff
@@ -163,13 +163,13 @@ public class OTRYapViewHandler: NSObject {
             return
         }
         
-        var sectionChanges:NSArray? = nil
-        var rowChanges:NSArray? = nil
+        let sectionChanges:AutoreleasingUnsafeMutablePointer<NSArray>? = nil
```

Comment:
This is what's needed to match the generated API https://developer.apple.com/reference/swift/autoreleasingunsafemutablepointer